src/neo/SmartContract/ApplicationEngine.cs

```diff
@@ -70,6 +71,32 @@ public ExecutionContext LoadScript(Script script, CallFlags callFlags, int rvcou
             return context;
         }
 
+        private StackItem ConvertReturnValue(object value)
```

Comment:
I prefer to return the values as VM types in order to skip this conversion.

Reply:
The purpose of this PR is to enable native contracts to directly call SYSCALLs without having to push and pop the stack.

Reply:
But if all syscalls return always a `StackItem` we don't need a conversor.

Reply:
If all syscalls return always a `StackItem`, you still need to convert the value to `StackItem` before returning. I think there is not too much difference.